app/models/booking.rb

```diff
@@ -5,4 +5,15 @@ class Booking < ActiveRecord::Base
   	accepts_nested_attributes_for :passengers, reject_if: :all_blank, allow_destroy: true
      # validates :flight_id, presence: true
      # validates :passengers, presence: true
+  def self.paypal_url(paypal_path, id, cost, passenger)
```

Comment:
Unused method argument - `id`. If it's necessary, use `_` or `_id` as an argument name to indicate that it won't be used.<br>Inconsistent indentation detected.